com.avaloq.tools.ddk.check.core/metamodel/com/avaloq/tools/ddk/check/Check.xtext

```diff
@@ -164,5 +164,6 @@ FeatureCallID:
 enum SeverityKind:
   error='error' | warning='warning' | info='info' | ignore='ignore';
 
-enum TriggerKind:  // we avoid using CheckKind to simplify writing expressions...
+// we avoid using CheckKind to simplify writing expressions...
+enum TriggerKind:

```

Comment:
fine for me, just as information, this bug has been fixed in the upcomming Xtext 2.26